### PR TITLE
[Cookbook] [Deployment] Added note about Nginx

### PR DESCRIPTION
**Pull request by phansys**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3+ |
| Fixed tickets |  |

### CODE REVIEW

File: cookbook/deployment/heroku.rst [hunk 1]
```diff
@@ -106,6 +106,15 @@ directory of the application and add just the following content:
 
     web: bin/heroku-php-apache2 web/
 
+.. note::
+
+    Nginx is also available in Heroku. If you prefer to use it, you can create a
```
Comment (xabbuh) [inline cookbook/deployment/heroku.rst]: on Heroku

Comment (phansys) [inline cookbook/deployment/heroku.rst]: Thank you @xabbuh, fixed.

File: cookbook/deployment/heroku.rst [hunk 2]
```diff
@@ -106,6 +106,15 @@ directory of the application and add just the following content:
 
     web: bin/heroku-php-apache2 web/
 
+.. note::
+
+    Nginx is also available on Heroku. If you prefer to use it, you can create a
+    Nginx config file (``nginx_app.conf`` for instance) to point from Procfile as described in `Heroku docs`_:
```
Comment (xabbuh) [inline cookbook/deployment/heroku.rst]: I think we should reword the paragraph a bit like this:

> If you prefer to use Nginx, which is also available on Heroku, you can create a configuration file for it and point to it from your Procfile as described in the Heroku documentation.

What do you think @phansys?

Comment (phansys) [inline cookbook/deployment/heroku.rst]: This sounds better. Updated.

**Comment by xabbuh** [comment]
:+1: nice work @phansys

**Comment by phansys** [comment]
Should I rebase the PR and resolve the conflicts?

**Comment by xabbuh** [comment]
@phansys That would be nice.

**Comment by phansys** [comment]
@xabbuh, done. Thanks.

**Comment by weaverryan** [comment]
I love it! Thanks Javier!

**Pull request merged**